.github/workflows/coverity.yml

```diff
@@ -1,15 +1,38 @@
 name: Coverity scan
 on:
-  # run once daily at 00:30 UTC due to
+  # coverity_daily job runs once daily at 00:30 UTC due to
   # https://scan.coverity.com/faq#frequency
   schedule:
     - cron: "30 0 * * *"
+  pull_request_target:
+    branches: [master]
   workflow_dispatch:
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 jobs:
-  coverity:
+  coverity_daily:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Install dependencies
+      uses: ./.github/actions/install-dependencies
+
+    - name: Configure
+      uses: ./.github/actions/configure
+
+    - name: Execute and submit coverity scan
+      uses: vapier/coverity-scan-action@v1.8.0
+      with:
+        email: "sssd-maint@redhat.com"
+        token: ${{ secrets.COVERITY_SCAN_TOKEN }}
+        working-directory: x86_64
+
+  # only if PR contains 'coverity' label
```

Comment:
Do you mind moving this comment before `pull_request_target`? If you add a little bit more of context, like what you did for the daily run it will help explain the behaviour

Reply:
Updated, please check.

Reply:
Looks perfect. Thanks!